src/components/CalendarMonth.jsx

```diff
@@ -53,11 +55,26 @@ export function getModifiersForDay(modifiers, day) {
   return day ? Object.keys(modifiers).filter(key => modifiers[key](day)) : [];
 }
 
+export function getWeekNumberForWeek(week) {
+  if (week && Array.isArray(week)) {
```

Comment:
You seem to be doing a whole lot of defensive checks here that I'm not sure are necessary. `week` is always going to be of a v. specific form and I also don't know that you need to clone the moment objects or call startOf('week') in this way.